RecoBTag/DeepFlavour/interface/jet_features_converter.h

```diff
@@ -0,0 +1,21 @@
+#ifndef RecoBTag_DeepFlavour_jet_features_converter_h
+#define RecoBTag_DeepFlavour_jet_features_converter_h
+
+namespace deep {
+
+  template <typename JetType, typename JetFeaturesType>
+  void jet_features_converter( const JetType & jet, JetFeaturesType & jet_features) {
```

Comment:
is JetFeaturesType argument needed in the template?

Reply:
probably not, will remove unnecessary templates